evals/benchmark/benchmark.py

```diff
@@ -248,7 +248,7 @@ def run_service_test(example, service_type, service, test_suite_config):
         deployment_type,
         test_suite_config.get("service_ip"),
         test_suite_config.get("service_port"),
-        test_suite_config.get("namespace")
+        test_suite_config.get("namespace"),
```

Comment:
can we do benchmarking on chatqna in difference namespaces?

Reply:
I suppose not. We did not modify that part of the benchmark tool, so it should remain as it was.

Reply:
already supported by https://github.com/opea-project/GenAIEval/pull/160